SVProfilePictureViewer/Language.cs

```diff
@@ -91,5 +91,23 @@ internal static class Language {
         };
       }
     }
+
+    internal static string WarnTitle {
+      get {
+        return GameInfo.CurrentLanguage switch {
+          "it" => "Attenzione",
+          "en" or _ => "Warning"
+        };
+      }
+    }
+
+    internal static string WarnContent {
```

Comment:
Change  the name of this string to ImportWarning. While I doubt there may be other warnings in this plugin, it is a better description of this text if that even arises.